### ISSUE TITLE
Can font's be zoomed?

### ISSUE THREAD

**Issue by J-Kallunki** [vrk-kpa/suomifi-ui-components#248]
## The Problem

Font's should be set with `rem` to allow zooming but how to do that with modular styles when it is affected by `body {font-size}`?

https://mobile.twitter.com/aleciavogel/status/1191810854095708160
